### DISCUSSION TITLE
How do I get my PR Reviewed & Merged?

### DISCUSSION THREAD

**Discussion by tophep** [reduxjs/redux-toolkit#2963] [Community Recipes]
Hey all 👋 I've never contributed here before but I have a [tiny PR here](https://github.com/reduxjs/redux-toolkit/pull/2904) that has been up for a while. Would love any feedback on proper procedure to get this reviewed and merged. Thanks!

— Christophe

**Comment by markerikson** [comment]
Hi! Normally we try to get back to PRs faster, but after shipping RTK 1.9 a couple weeks ago we've been taking rest of the month of November off from meaningful maintenance work to recover and enjoy the holidays.  I'm hoping to get back into some RTK stuff in the next few days.  Thanks for the reminder!
